api/scpca_portal/test/test_metadata_file.py

```diff
@@ -19,13 +19,26 @@ def setUp(self):
                 "country": "Spain",
                 "language": "Spanish",
                 "capital": "Madrid",
+                "holidays": [
+                    "New Year’s Day",
+                    "Epiphany",
+                    "Good Friday",
+                    "Labor Day",
+                    "Assumption Day",
+                    "Hispanic Day",
+                    "All Saints’ Day",
+                    "Constitution Day",
+                    "Immaculate Conception Day",
+                    "Christmas Day",
```

Comment:
Could you reduce this list to 2 or 3 items. The rest are superfluous so should be fine to omit them.

Reply:
See the commit at aff961cae29896428fdd0fa76b1ac61dfa35a2bf.